domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/iterableAssertions.kt

```diff
@@ -80,6 +79,14 @@ fun <E : Any> _hasNotNext(expect: Expect<Iterable<E>>): Assertion =
         !it.iterator().hasNext()
     }
 
+fun <E : Comparable<E>, T : Iterable<E>> _min(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, E> =
+    ExpectImpl.feature.extractor(assertionContainer)
+        .methodCall("min")
+        .withRepresentationForFailure(DescriptionIterableAssertion.NO_ELEMENTS)
+        .withCheck { it.iterator().hasNext() }
+        .withFeatureExtraction { it.min()!! }
```

Comment:
@robstoll please check this.

Reply:
also the signature of this method.

Reply:
Signature is fine
I would rewrite `!!` to `?: throw IllegalStateException("Iterable does not haveNext even though checked before. Concurrent access?")`

What else shall I check?

Reply:
done. just the non compiling test. It requires a val. should I implement the val in the api?